src/App.vue

```diff
@@ -1,60 +1,39 @@
 <template>
   <v-app>
-    <v-app-bar
-      app
-      color="primary"
-      dark
-    >
-      <div class="d-flex align-center">
-        <v-img
-          alt="Vuetify Logo"
-          class="shrink mr-2"
-          contain
-          src="https://cdn.vuetifyjs.com/images/logos/vuetify-logo-dark.png"
-          transition="scale-transition"
-          width="40"
-        />
-
-        <v-img
-          alt="Vuetify Name"
-          class="shrink mt-1 hidden-sm-and-down"
-          contain
-          min-width="100"
-          src="https://cdn.vuetifyjs.com/images/logos/vuetify-name-dark.png"
-          width="100"
-        />
-      </div>
-
-      <v-spacer></v-spacer>
-
-      <v-btn
-        href="https://github.com/vuetifyjs/vuetify/releases/latest"
-        target="_blank"
-        text
-      >
-        <span class="mr-2">Latest Release</span>
-        <v-icon>mdi-open-in-new</v-icon>
-      </v-btn>
-    </v-app-bar>
-
+    <headerContent class="mx-lg-auto my-5 my-lg-10 px-5 pt-5"/>
     <v-main>
-      <HelloWorld/>
+      <mainContent class="mx-auto mb-10"/>
     </v-main>
   </v-app>
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld';
+import headerContent from './components/headerContent.vue';
+import mainContent from './components/mainContent.vue';
 
 export default {
   name: 'App',
 
   components: {
-    HelloWorld,
+    headerContent,
+    mainContent
   },
 
   data: () => ({
     //
   }),
 };
 </script>
+
+<style>
+.v-data-table > .v-data-table__wrapper > table > thead > tr > th {
+   font-size: 1rem !important;
+}
+
+.v-data-table > .v-data-table__wrapper > table > tbody > tr > td {
+   font-size: 0.75rem !important;
+   margin: 0;
+   padding-right: 0;
+   line-height: 1;
+}
+</style>
```

Comment:
u can also set `lang=scss` if u want to use SASS which is what we usually go for

```css
<style lang="scss">
  .v-data-table {
    td  {
      font-size: 1rem !important;
    }
  }
  ...
</style>
```